rl_studio/config/config.yaml

```diff
@@ -85,6 +85,37 @@ agent:
         penal: 0
         min_reward: 1_000
         highest_reward: 100
+  autoparking:
```

Comment:
This is something we may modify like we commented via slack.
We should make the config as generic as possible so we can indicate the parameters we want to run to our agent without need to indicate what agent we run.
I mean, instead of
f1:
camera ...
reward ...
autoparking:
camera ...
reward ....

We could have:
agent:
camera ...
reward ...

And just replace according to our preferences.

Let me what you think. It is just an idea.

Reply:
yes, we have to think about how to better config.yaml